demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -454,73 +454,68 @@ def disable_md_autolinks(markdown: str) -> str:
     return re.sub(r'\b(?<!\)\[)(https?)://([\w\d]+?\.[\w\d]+?)\b', r'\1:<span>//</span>\2', markdown, flags=re.IGNORECASE)
 
 
-def generate_command_example(cmd, cmd_example=None):
+def generate_command_example(cmd_from_yaml, cmd_example=None):
+    example = []
     errors = []
-    context_example = None
-    md_example = ''
-    if cmd_example is not None:
-        cmd_example, md_example, context_example = cmd_example
-    else:
-        cmd_example = ' '
-        errors.append('did not get any example for {}. please add it manually.'.format(cmd['name']))
+    if not cmd_example:
+        errors.append(f'did not get any example for {cmd_from_yaml["name"]}. please add it manually.')
 
-    example = [
-        '',
-        '#### Command Example',
-        '```{}```'.format(cmd_example),
-        '',
-    ]
-    if context_example and context_example != '{}':
+    else:
         example.extend([
-            '#### Context Example',
-            '```json',
-            '{}'.format(context_example),
-            '```',
+            '',
+            '#### Command',
+            '```{}```'.format(cmd_from_yaml['name']),
```

Comment:
are you sure this part is needed? I don't see it in the previous implementation.
Please show me offline an example output.